### ISSUE TITLE
bip-taproot: Motivation section doesn't address motivation clearly

### ISSUE THREAD

**Issue by AdamISZ** [sipa/bips#138]
Discussion group 6 agreed largely with these notes I'd made about this section of the BIP; I would PR but I don't think that's appropriate, since it's kind of central to what the author(s) are writing:

--------------------------

*Do the goals make sense?*

I don't like that the Motivation section of the BIP talks about the Motivation for not adding in other goals. It should just state the goal of the BIP, instead. This is context, which many fresh readers will not have and it's unreasonable to expect them to read everything about graftroot in order to understand the motivation of the taproot BIP.

The actual goal of this BIP afaict is only in the first line of the "Design" section:

```
This proposal focuses on improvements to privacy, efficiency, and flexibility of Bitcoin's smart contracts, subject to two restrictions:

Not adding any new strong security assumptions
Not combining into the proposal any functionality which could be simply implemented independently.
```

The following "Specifically" section could additionally mention the support for much larger scripts than previously (the other huge change along with the mentioned Privacy changes).

The following bullet points I like, they clarify at a high level each of the functional effects of the new design.

Another point on the Design section: it would surely make sense to write the approximate structure of the scriptPubkey (Q = P + H(P||S)G) here for those who want to understand in broad terms, instead of requiring them to read the detailed algorithm in "Script Validation Rules". Perhaps debatable, but it seems that way to me.

**Comment by sipa** [comment]
Here is a suggested change that addresses most of the comments: https://github.com/sipa/bips/pull/181. I have not included the larger script sizes as I think that belongs more in bip-tapscript.

**Comment by sipa** [comment]
Fixed by #181.
